rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/CommandTableQueryFactory.java

```diff
@@ -100,7 +104,8 @@ public SetOkStatusQuery newSetOkStatusQuery(String id) {
         final SetOkStatusQuery.Builder builder = SetOkStatusQuery.newBuilder()
                                                                  .setDataSource(dataSource)
                                                                  .setLogger(logger)
-                                                                 .setIdColumn(idColumn)
+                                                                 .setIdColumn(
```

Comment:
Please see if it can fit into one line. If not, consider having the whole builder starting from a new line.